terraform/scripts/mongo_disk_setup.yml

```diff
@@ -11,7 +11,7 @@ mounts:
 
 bootcmd:
   # Wait for EBS disks to be attached
-  - 'while [ `lsblk | wc -l` -lt 6 ]; do echo Waiting for disks to attach...; sleep 5; done'
+  - 'while [ `lsblk | grep -c " disk"` -lt 4 ]; do echo Waiting for disks to attach...; sleep 5; done'
```

Comment:
I asked @dav3r if this change was intentional, and he said it was.  When he booted up in Jessie the root volume had two partitions, which broke the old logic.  He amended the logic to work in both cases.